dashboard/app/models/plc/enrollment_module_assignment.rb

```diff
@@ -28,17 +29,16 @@ class Plc::EnrollmentModuleAssignment < ActiveRecord::Base
       COMPLETED = :completed
   ]
 
-  # The module status is completely determined by the completeness of its task assignments
   def status
-    tasks_to_complete = plc_task_assignments.joins(:plc_task).where.not('plc_tasks.type': 'Plc::LearningResourceTask')
-    completed_tasks = tasks_to_complete.where(status: Plc::EnrollmentTaskAssignment::COMPLETED)
+    levels_tracked = UserLevel.where(user: user, level: plc_learning_module.stage.script_levels.map(&:levels).flatten)
```

Comment:
Nit: can use `.map(&:level_id)` instead of `map(&:levels).flatten` since only the ID is needed for the next line.